modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -577,6 +640,73 @@ public CompletableFuture<Boolean> startReplica(
         }
     }
 
+    /**
+     * Starts a replica. If a replica with the same partition id already exists, the method throws an exception.
+     *
+     * @param replicaGrpId Replication group id.
+     * @param listener Replica listener.
+     * @param snapshotStorageFactory Snapshot storage factory for raft group option's parameterization.
+     * @param newConfiguration A configuration for new raft group.
+     * @param raftGroupListener Raft group listener for raft group starting.
+     * @param raftGroupEventsListener Raft group events listener for raft group starting.
+     * @throws NodeStoppingException If node is stopping.
+     * @throws ReplicaIsAlreadyStartedException Is thrown when a replica with the same replication group id has already been
+     *         started.
+     */
+    public CompletableFuture<Replica> startReplica(
```

Comment:
Is it a merge artifact? Why do we need this additional method to ones introduces by Misha?

Reply:
Nope, this method have no storageIndexTracker for example and witll not have it in future, I think.